### ISSUE TITLE
Update changelog tool for https://github.com/nodejs/node/pull/6503

### ISSUE THREAD

**Issue by jasnell** [nodejs/nodejs.org#733]
The `scripts/release-post.sh` tool will need to be updated to understand the new changelog structure once https://github.com/nodejs/node/pull/6503 lands. The new PR moves the specific changelogs for each release line into separate files in `doc/changelogs/*`. The pattern is simple: v4 releases will go into `doc/changelogs/CHANGELOG_V4.md`, v5 releases go into `doc/changelogs/CHANGELOG_V5`, etc. The specific changelog entries will no longer be listed in the root `CHANGELOG.md` file.

**Comment by evanlucas** [comment]
I've been meaning to do this. I ended up just doing v6.2.0 manually. I'll try to get to it in the next few days though.

**Comment by evanlucas** [comment]
Ah, scratch that, didn't realize @lpinca had already started on it. Thanks!!!!
